tests/cases/fourslash/jsxTagLinkedEdit7.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path='fourslash.ts' />
+
+// @FileName: /fragment.tsx
+////const jsx = (
+////    /*7*/</*0*/>/*1*/
+////        <img />
+////    /*8*/</*2*///*3*/>/*4*/
+////);
+////const jsx2 = (
+////    <>/*5*/
+////    </>
+////);
+////const jsx3 = (
+////    </*6*/>
+////        <div>
+////    </>
+////);
+
+const linkedCursors7 = {ranges: [{start: 14, end: 14}, 
+                            {start: 43, end: 43}],
+                        wordPattern : undefined};

```

Comment:
The way testing works right now is not ideal because we can't tell where positions 14 and 43 are. Instead, consider just handing over an array of marker names.